### ISSUE TITLE
Include ENVO subsets?

### ISSUE THREAD

**Issue by pbuttigieg** [EBISPOT/OLS#154]
Greetings OLS Team,

We have a bunch of ENVO subsets that have been requested by various communities (a little documentation [here](https://github.com/EnvironmentOntology/envo/wiki/Subsets)). It would be very useful for them if they were able to browse these via the OLS, rather than having to search through  ENVO's full graph.

Subsets are identified by an `in subset` annotation property in our [primary OWL file,](http://purl.obolibrary.org/obo/envo.owl) but we export these as separate OWL artifacts. You can find the complete list of exports [here](https://github.com/EnvironmentOntology/envo/tree/master/subsets).

Each subset OWL file has it's own PURL in the form: 
`http://purl.obolibrary.org/obo/envo/subsets/[subsetName].owl`

For example
* http://purl.obolibrary.org/obo/envo/subsets/envoOmics.owl (perhaps of particular interest to some EBI users)
* http://purl.obolibrary.org/obo/envo/subsets/envoPolar.owl
* http://purl.obolibrary.org/obo/envo/subsets/environmental_hazards.owl
* http://purl.obolibrary.org/obo/envo/subsets/nlcd2011.owl

Thinking about automatic updating: These PURLs can be parsed out of each OWL file by looking at the `xml:base` value in the `rdf:RDF' block or the `rdf:about` value in the `owl:Ontology` block. For example, [here](https://github.com/EnvironmentOntology/envo/blob/master/subsets/envoPolar.owl#L3), and [here](https://github.com/EnvironmentOntology/envo/blob/master/subsets/envoPolar.owl#L18) respectively.

Would it be possible to add these and associate with the main ENVO instance in the OLS?

**Comment by simonjupp** [comment]
Yes, we have support for detecting and indexing the in_subset property and we plan to expose this through the UI soon. You'll be able to view and browse subsets without us needing to load the actual subset files. We don't have a timeline on when this will be released but I'll put any updates in this ticket.

**Comment by joncison** [comment]
Glad to hear!  EDAM will benefit from this too.  

I currently define subsets using ``<oboInOwl:inSubset>`` : is that good ?

**Comment by pbuttigieg** [comment]
>Yes, we have support for detecting and indexing the in_subset property and we plan to expose this through the UI soon. You'll be able to view and browse subsets without us needing to load the actual subset files. We don't have a timeline on when this will be released but I'll put any updates in this ticket.

Fantastic! This will help focus communities on resources for their domain. Will this issue be the place to watch for updates?

**Comment by pbuttigieg** [comment]
> I currently define subsets using <oboInOwl:inSubset> : is that good ?

Same here: `http://www.geneontology.org/formats/oboInOwl#inSubset`

**Comment by joncison** [comment]
Thanks :)

**Comment by cmungall** [comment]
Is this ticket done, or is there an additional request to have subset views? It would be great to dynamically filter by subset, have subsets colored

**Comment by henrietteharmse** [comment]
Hi Chris, this is not done yet, but it is in the pipeline.
